__tests__/cache-restore.test.ts

```diff
@@ -102,6 +103,9 @@ describe('cache-restore', () => {
 
     // cache-utils
     getCommandOutputSpy = jest.spyOn(utils, 'getCommandOutput');
+
```

Comment:
Is this change and L53 change is required? Because I think in restore restoreCache function we don't check ``isCacheActionAvailable`` functions. This we are checking in main.ts

Reply:
Done. Thank you.